# HG changeset patch
# User Rik <rik@octave.org>
# Date 1413907067 25200
#      Tue Oct 21 08:57:47 2014 -0700
# Node ID 66ca15a97988ca0b6612146bce696369b75e0dbb
# Parent  dc64f1c4a33e97a7308800054c028ba109b0a9e8
doc: Remove deprecated dump_prefs function from the manual.

* basics.txi: Remove dump_prefs from manual.

* toplev.cc (Foctave_config_info): Removee seealso link to dump_prefs.

diff --git a/doc/interpreter/basics.txi b/doc/interpreter/basics.txi
--- a/doc/interpreter/basics.txi
+++ b/doc/interpreter/basics.txi
@@ -372,21 +372,16 @@ Octave searches for @file{.octaverc}.
 If you start Octave in your home directory, commands from the file
 @file{~/.octaverc} will only be executed once.
 @end table
 
 A message will be displayed as each of the startup files is read if you
 invoke Octave with the @option{--verbose} option but without the
 @option{--silent} option.
 
-The @code{dump_prefs} function is useful for determining what customizations
-to Octave are possible and which are in effect.
-
-@DOCSTRING(dump_prefs)
-
 @node Quitting Octave
 @section Quitting Octave
 @cindex exiting octave
 @cindex quitting octave
 @cindex finish.m
 @cindex site exiting file
 
 Shutdown is initiated with the @code{exit} or @code{quit} commands (they are
diff --git a/libinterp/corefcn/toplev.cc b/libinterp/corefcn/toplev.cc
--- a/libinterp/corefcn/toplev.cc
+++ b/libinterp/corefcn/toplev.cc
@@ -1246,17 +1246,17 @@ DEFUN (octave_config_info, args, ,
 @deftypefn  {Built-in Function} {} octave_config_info ()\n\
 @deftypefnx {Built-in Function} {} octave_config_info (@var{option})\n\
 Return a structure containing configuration and installation information for\n\
 Octave.\n\
 \n\
 If @var{option} is a string, return the configuration information for the\n\
 specified option.\n\
 \n\
-@seealso{dump_prefs, computer}\n\
+@seealso{computer}\n\
 @end deftypefn")
 {
   octave_value retval;
 
 #if defined (ENABLE_DYNAMIC_LINKING)
   bool octave_supports_dynamic_linking = true;
 #else
   bool octave_supports_dynamic_linking = false;
